udfs/community/day_occurrence_of_month.sql

```diff
@@ -0,0 +1,29 @@
+  
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+-- day_occurrence_of_month: Returns the n th occurrence of the weekday in the month for the specified date.
+-- Input: date_expression DATE or TIMESTAMP
+-- Output: The result is an INTEGER value between 1 and 5, representing the n th occurrence of the weekday in the month.
+CREATE OR REPLACE FUNCTION fn.day_occurrence_of_month(date_expression ANY TYPE) AS 
+(
+  (
+    SELECT 
+      IF(EXTRACT(DAYOFWEEK FROM DATE_TRUNC(CAST(date_expression AS DATE), MONTH)) > EXTRACT(DAYOFWEEK FROM date_expression),
+        fn.week_number_of_month(date_expression),
+        fn.week_number_of_month(date_expression) + 1)
+  )
+);
```

Comment:
Add trailing new line

Reply:
Where to add a trailing new line? I don't see any issues with formatting.